thirdparty/SA/libdivsufsort/include/config.h

```diff
@@ -0,0 +1,81 @@
+/*
+ * config.h for libdivsufsort
+ * Copyright (c) 2003-2008 Yuta Mori All Rights Reserved.
+ *
+ * Permission is hereby granted, free of charge, to any person
+ * obtaining a copy of this software and associated documentation
+ * files (the "Software"), to deal in the Software without
```

Comment:
All the `thirdparty/SA/` code seems to be copied directly from [libdivsufsort](https://github.com/y-256/libdivsufsort). Unless there is a good reason not to, I think it would be best to add this to our list of dependencies and import it. This avoids us having our own copies to maintain.

Reply:
This is a good suggestion. I am not familar with the dependency mechanism and I will learn how to do it.